worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -810,6 +810,19 @@ namespace RTC
 			return;
 		}
 
+		// If the packet belongs to current spatial layer being sent and packet does
+		// not have payload other than padding, then drop it.
+		if (spatialLayer == this->currentSpatialLayer && packet->GetPayloadLength() == 0)
+		{
+			this->rtpSeqManager.Drop(packet->GetSequenceNumber());
+
+#ifdef MS_RTC_LOGGER_RTP
+			packet->logger.Dropped(RtcLogger::RtpPacket::DropReason::EMPTY_PAYLOAD);
+#endif
+
+			return;
+		}
+
```

Comment:
This block should be removed https://github.com/versatica/mediasoup/blob/8b1ba663caec7b0a6909717e251f4a80e2f4397b/worker/src/RTC/SimulcastConsumer.cpp#L734

Reply:
Upps, yes